src/main/resources/openapi/audio-requests.yaml

```diff
@@ -0,0 +1,96 @@
+openapi: 3.0.1
+
+servers:
+  - url: http://localhost:4550/
+
+info:
+  description: Modernised DARTS (Digital Audio Recording and Transcription Service).
+  version: "0.0.1"
+  title: Modernised DARTS
+
+paths:
+  /audio-requests:
+    get:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Audio Requests
+      summary: Users can view a summary of their audio requests and status
+      operationId: getYourAudio
+      description: '"Your audio" requests for a user to view whether Current / Expired'
```

Comment:
`Your Audio` is somethgin on the screen, i think `Audio requests....` would be sufficient?